app/lib/forms/choose_your_provider.rb

```diff
@@ -83,5 +81,18 @@ def validate_lead_provider_exists
         errors.add(:lead_provider_id, :invalid)
       end
     end
+
+    def email_template
+      return :not_england_wrong_catchment if wizard.store["funding_eligiblity_status_code"] == Services::FundingEligibility::NOT_IN_ENGLAND
```

Comment:
Question: Pattern `wizard.store["funding_eligiblity_status_code"] == Services::FundingEligibility::NOT_IN_ENGLAND` repeats several times in this PR, would it make sense to have helper methods like `wizard.application_from_england?` instead?

Reply:
Yeah im happy for that!